next.config.js

```diff
@@ -9,6 +13,16 @@ const nextConfig = {
     config.node = {
       fs: 'empty',
     };
+    config.module.rules.push({
```

Comment:
I think what this rule is unnecessary, because fonts can be cached by browser (and it's better for my opinion for performance), and inline size for images can be configured by `next-images` by [inlineImageLimit ](https://www.npmjs.com/package/next-images#inlineimagelimit) option. Also, in production build webpack adds an unique hash to filenames, in this case the name will be always the same after each build